### DISCUSSION TITLE
GUI scaling in tall aspect ratios

### DISCUSSION THREAD

**Discussion by KarstenTWolf** [FakeFishGames/Barotrauma#13686] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When launching the game in tall aspect ratios like 16:18 (2560x2880), the GUI scaling is way off. Ingame containers have extra buffers, the text is quite large and the chat interface is nearly unusable. Sliding the settings controls helps in menus and things like mission popups but text in the fabricator for instance don't scale. 

Image example: https://ibb.co/sbwsLgM

### Reproduction steps

Start game in 16:18 aspect ratio.
Play missions, host servers, do anything. Game doesn't know what to do with this edge case. 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.0.21.0

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report!

I wasn't even aware the game supports aspect ratios like this. Is there some reason to use such an odd aspect ratio?

**Reply by jigardi** [reply]
I'm not sure if this is due to the same issue or not, but the F3 console incorrectly formats its output at 4K resolution - no matter the display mode.
![image](https://github.com/user-attachments/assets/84a1717e-3ea9-4d29-a53e-13aca8c6889d)
^In the image I first ran the command at 1920x1080 windowed mode, then changed to fullscreen 4K and ran it again. You can see that the results from the first run of the "help" command still display well, even after the resolution change.

**Comment by KarstenTWolf** [comment]
The tiny minority of players that have a LG Dual Up screen, which might just be me. 

https://www.lg.com/us/monitors/lg-28mq780-b-dualup-monitor

**Comment by jigardi** [comment]
I can confirm this is also the case in 4K.

<img width="925" height="310" alt="image" src="https://github.com/user-attachments/assets/b600f2c8-1b02-4f93-92d5-ff2e9500cc63" />
I have a secondary screen on the side running at 1080x1920.

Edit:  Just noticed that I replied to this back in 2025.  Whops my bad : S
Edit2: Clarity.  Accidentally pasted the edit into the middle of the post.
